ambry-network/src/main/java/com.github.ambry.network/SocketServer.java

```diff
@@ -325,13 +320,8 @@ public void run() {
   /*
    * Create a server socket to listen for connections on.
    */
-  private ServerSocketChannel openServerSocket(String host, int port) throws IOException {
-    InetSocketAddress address = null;
-    if (host == null || host.trim().isEmpty()) {
-      address = new InetSocketAddress(port);
-    } else {
-      address = new InetSocketAddress(host, port);
```

Comment:
@xiahome This should be the only place (for production code, at least) we need to change this. On the frontend, we Netty already does this when we call `ServerBootstrap.bind(port)`.